ant-networking/src/driver.rs

```diff
@@ -684,7 +681,6 @@ impl NetworkBuilder {
             request_response,
             kademlia,
             identify,
-            #[cfg(feature = "local")]
             mdns,
```

Comment:
This is where the mDNS behavior is executed both with and withouth the `--local` flag.